tests/jest/data/package.json

```diff
@@ -0,0 +1 @@
+/Users/mh015940/source/kaiju-plugin-utils/package.json
```

Comment:
![image](https://user-images.githubusercontent.com/6720991/34399746-2072de1e-eb50-11e7-8748-1f99d60a5ebe.png)

Reply:
That does look shady, this is supposed to be a symlink. I'm not sure how well files like this translate into git, but the test using the file still passes. Which might be an indicator of the quality of test. 

https://github.com/cerner/kaiju-plugin-utils/pull/1/files#diff-81255a28f27e8cd84fb6ab3fc0eceffaR13